File: 3roC/Ejercicio1.cpp
#include <iostream>

using namespace std;

int main()
{
  // esto es un comentario de línea. La computadora no ejecuta el codigo luego de encontrar "//". Tambien existen los comentarios multilinea, con los caracteres "/*" para abrir, y los caracteres "*/" para terminar. Ejemplo:

  /* Este es un comentario multilínea,
  y la computadora lo ignora. :)
  */

  int numero1; // declaramos la variable numero1, de tipo numérico entero
  int numero2; // declaramos la variable numero2, de tipo numérico entero
  int suma, resta, producto; // también podemos declarar varias variables de un mismo tipo en una sola línea
  cout << "Ingresa dos números enteros por favor:" << endl; // mostramos mensaje por pantalla. endl es un "enter", para saltar de linea

  // leemos los números ingresados por el usuario, y los guardamos en las variables numero1 y numero2
  cin >> numero1;
  cin >> numero2;

  suma = numero1 + numero2;
  resta = numero1 - numero2;
  producto = numero1 * numero2;

  cout << "Suma: " << suma<<endl;
  cout << "Resta" << resta<<endl;
  cout << "Producto" << producto<<endl;

  // Nos falta mostrar la division, lo hacemos en clase :)

  return 0;
}

File: 6toB/Ejercicios/ejercicio1.js
// Esto es un comentario de linea. La computadora ignora todo este texto, no lo ejecuta

// despues tenemos el comentario multilinea, se escribe con /* */. Ejemplo:

/* Esto es un comentario multilinea

let numero1 = prompt("Ingrese un numero por favor");
let numero2 = prompt("Ingrese otro numero por favor");

El problma que teniamos en clase con estas dos lineas de codigo es que el dato que ingresamos usando prompt() es una cadena de caracteres (string). Tenemos que "castearlo" a tipo de dato numérico.
*/
let numero1 = parseInt(prompt("Ingrese un numero por favor")); // declaramos e inicializamos la variable con el valor casteado usando parseInt()

let numero2 = prompt("Ingrese otro numero por favor"); // declaramos una variable para guardar el numero2, que en este momento es un string. Por ejemplo, si ingresamos 10, entonces: numero2 <= "10"
numero2 = parseInt(numero2); // parseInt() castea el dato pasado entre los parentesis a tipo de dato numérico entero
// numero2 <= 10, en este ejemplo :)

let suma = numero1 + numero2;
let resta = numero1 - numero2;
let producto = numero1 * numero2;

if (numero2 != 0) {
  let division = numero1 / numero2;
  alert("Division: " + division);
}

alert("Suma: " + suma);
alert("Resta: " + resta);
alert("Producto: " + producto);